.github/issue_template.md

```diff
@@ -50,9 +50,7 @@ logs.
 
 ```Template for output < 10 lines```
 
-1. Client logfile: Output of `owncloud --logwindow` or `owncloud --logfile log.txt`
-(On Windows using `cmd.exe`, you might need to first `cd` into the ownCloud directory)
-(See also https://doc.owncloud.org/desktop/troubleshooting.html#log-files )
+1. Client [logfile](https://doc.owncloud.com/desktop/latest/appendices/troubleshooting.html#log-files)
```

Comment:
Maybe we can make it shorter and hopefully more robust for future changes:

```suggestion
1. Client [logfile](https://doc.owncloud.com/desktop/troubleshooting.html#log-files)
```

Reply:
I didn't use the link as the #log-files does not work.